st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -321,14 +323,30 @@ def get_one(self, id, output_type=None, requester_user=None):
         if output_type and output_type != 'all':
             query_filters['output_type'] = output_type
 
+        def format_output_object(output_db_or_api):
+            if output_format == 'raw':
+                return output_db_or_api.data
+            elif output_format == 'event_source':
+                if not isinstance(output_db_or_api, ActionExecutionOutputAPI):
+                    data = ActionExecutionOutputAPI.from_model(output_db_or_api)
+                else:
+                    data = output_db_or_api
+
+                event = 'st2.execution.output__create'
+                result = 'event: %s\ndata: %s\n\n' % (event, json_encode(data, indent=None))
+                return result
```

Comment:
We can add a third option that does not serialize json. I believe that you can also send timestamp as `id` property of eventsource package.

Reply:
As discussed yesterday, we can just leave it like that for now and if turns out that it's too slow, I can change it.

What do you think?